templates/job-submitted.php

```diff
@@ -35,6 +35,30 @@
 				get_permalink( $job->ID )
 			)
 		);
+
+		$permalink = job_manager_get_permalink('job_dashboard');
+		$title = get_the_title((job_manager_get_page_id('job_dashboard')));
```

Comment:
PHPCS: Align equal sign with one above with spaces and remove extra parens.

```suggestion
		$title     = get_the_title( job_manager_get_page_id( 'job_dashboard' ) );
```